types/types.go

```diff
@@ -466,6 +466,18 @@ type ImageInspectInfo struct {
 	Os            string
 	Layers        []string
 	Env           []string
+	LayersDetail  []LayerDetail
+	History       []v1.History
+	Author        string
+	Size          int64
+	Config        v1.ImageConfig
+}
+
+type LayerDetail struct {
```

Comment:
`ImageInspectLayer`?

Reply:
I think ImageInspectLayer  and  Layers are same meaning, at .ImageInspectInfo struct

Reply:
The `types` subpackage contains quite a few types … and it’s useful to make it clear that the purpose of this struct is _only_ as an inspect output.

We already have `types.BlobInfo` (and `manifest.LayerInfo`); being clear about the purposes of each (especially when there is no written documentation) is helpful to users of the API.

(Especially when may users are just going to use `Inspect().LayersDetail…` without having to name the type, we need the type name more to be unambiguous than we need it to be short.)